src/common/aes/aes.c

```diff
@@ -11,44 +11,55 @@
 
 #if defined(OQS_DIST_X86_64_BUILD)
 #define C_OR_NI_OR_ARM(stmt_c, stmt_ni, stmt_arm) \
-    if (OQS_CPU_has_extension(OQS_CPU_EXT_AES)) { \
-        stmt_ni; \
-    } else { \
-        stmt_c; \
-    }
+   do { \
+      if (OQS_CPU_has_extension(OQS_CPU_EXT_AES)) { \
+          stmt_ni; \
+      } else { \
+          stmt_c; \
+      } \
+   } while(0)
```

Comment:
This is a personal style choice. It basically forces the user to add a `;` after the macro so it looks more like a function call. I changed aes to be like this as that's how I wrote the sha2 version originally and wanted to make the style consistent between them

Reply:
It's also worth mentioning that I've seen this in several places before such as FreeBSD's `list.h` header. The compiler optimizes out the guard check since it will always be false.

Reply:
Okay, makes sense.